p2p/conn/connection.go

```diff
@@ -646,7 +646,7 @@ FOR_LOOP:
 			}
 		default:
 			err := fmt.Errorf("unknown message type %v", reflect.TypeOf(packet))
-			c.Logger.Error("Connection failed @ recvRoutine", "conn", c, "err", err)
+			c.Logger.Debug("Connection failed @ recvRoutine", "conn", c, "err", err)
```

Comment:
This should probably remain an error, since we received an unknown message type.